app/frontend/actions.js

```diff
@@ -201,12 +201,12 @@ module.exports = ({setState, getState}) => {
   }
 
   const openAddressDetail = (state, {address, bip32path}) => {
-    const showAddressVerification = state.usingTrezor
+    const showAddressVerification = state.usingTrezor && bip32path
```

Comment:
I would explain in a comment that this is done to avoid  trezor checking for donation address